docs/articles/documentation/test-api/selecting-page-elements/dom-node-snapshot-members.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: docs
+title: DOM Node Snapshot Members
+permalink: /documentation/test-api/selecting-page-elements/dom-node-snapshot-members.html
+---
+# DOM Node Snapshot Members
```

Comment:
Maybe just `DOM Node Snapshot`? Members is overlapping in side menu and looks like a separate menu item